capabilityreceiver/cmd/migratevalidations/main.go

```diff
@@ -186,18 +157,15 @@ func addToValidationTable(ctx context.Context, args *map[string]interface{}) err
 
 	for _, val := range validationVals {
 		// Create the capability statement object
-		var capStat capabilityparser.CapabilityStatement
-		var capInt map[string]interface{}
-		if len(val.capStatByte) > 0 {
-			err = json.Unmarshal(val.capStatByte, &capInt)
-			if err != nil {
-				log.Warnf("Error while unmarshalling the rows of the history table for URL %s. Error: %s", wa.fhirURL, err)
-			}
-
-			capStat, err = capabilityparser.NewCapabilityStatementFromInterface(capInt)
-			if err != nil {
-				log.Warnf("unable to parse CapabilityStatement out of message with url %s. Error: %s", wa.fhirURL, err)
-			}
+		capStat, err := capabilityparser.NewCapabilityStatement(val.capStatByte)
+		if err != nil {
+			log.Warnf("unable to parse CapabilityStatement out of message with url %s. Error: %s", wa.fhirURL, err)
+		}
```

Comment:
This line might need to change due to the changes I made in my default fhir version PR. Maybe a @todo should be added here? Basically the changes in that PR make it so that sometimes a custom error is returned saying that no capability version match was found, and that it is defaulting to DSTU2 instead. Therefore, there will still be a capStat returned even though the error is not nil. The solution I found was to also check if the capstat is nil as well as if the error is not nil. Just something I thought of based on recent changes made!